src/js/view/preview.js

```diff
@@ -12,6 +12,11 @@ function validState(state) {
 Object.assign(Preview.prototype, {
     setup: function(element) {
         this.el = element;
+        this.hasZoomThumbnail = !this.model.get('autostart') && this.model.get('_abZoomThumbnail');
```

Comment:
`model.get('autostart')` doesn't really mean we will or won't autostart. I would ignore that and just let state changes and `setImage` (plus the ab flag) determine if we should animate-zoom the image or not.